Add supportlevelheader to partner collectors (#30084)
demisto/content: Packs/AbnormalSecurity/ReleaseNotes/2_1_1.md

#### Integrations

##### Abnormal Security Event Collector

> - Added the **supportlevelheader** key to specify that the integration is supported by Palo Alto Networks in the description file.

Reviewer: ```suggestion
- Added a notice to the documentation that this integration is supported by XSOAR.
```
